### PR TITLE
Makes changes suggested in #94 and #95

### LINKED ISSUE #95: Refactor fix-tests branch with Nelsons comments
- [x] [inline requires -> reduces lines of code and number of vars being set.](https://github.com/emfoundation/ce100-app/pull/64#discussion_r77640484)
- [x] [npm uninstall hashids](https://github.com/emfoundation/ce100-app/pull/64#discussion_r77637979) 
- [x] [npm uninstall codecov](https://github.com/emfoundation/ce100-app/pull/64#discussion_r77638568)
- [x] [worth adding the travis badge and david-dm badges too](https://github.com/emfoundation/ce100-app/pull/64#discussion_r77637306)  see https://github.com/dwyl/repo-badges
- [x] [Remove server/start.js](https://github.com/emfoundation/ce100-app/pull/64#discussion_r77637306)
- [x] [Remove all consts](https://github.com/emfoundation/ce100-app/pull/64#discussion_r77641302)

### PR DESCRIPTION
**Pull request by mk4111**
Closes #95.
Need to split functions in handlers - pending on answer to #110 before doing so.

# READY TO REVIEW (PLEASE!)

### CODE REVIEW

**Comment by mk4111** [comment]
Hi @nelsonic, this branch makes the changes that you've suggested.
Everything should be there minus the splitting out handlers.js into different files as you've requested because I thought that it might be easier to compare the differences this way (...retrospective thought - I'm really sorry for assuming on your behalf...). I hope this is okay...!

**Comment by nelsonic** [comment]
@mk4111 please fix merge conflicts again. thanks. 👍

**Comment by mk4111** [comment]
Hi @nelsonic! I've fixed the merge conflicts, could you please have a look? Thank you!

**Comment by nelsonic** [comment]
@mk4111 I _started_ trying to resolve the merge conflicts but got nowhere... 😕 
I can _push_ up my changes to GH but I _suspect_ you might be quicker at resolving the conflicts...
let me know your preference.

**Comment by mk4111** [comment]
Hi @nelsonic! This should now be ready for review. It should close #94 and #95 too!

**Comment by JMurphyWeb** [comment]
@mk4111 the readme looks like it has a merge hangover in it?

**Comment by mk4111** [comment]
@JMurphyWeb - good spot! Thank you!

**Comment by nelsonic** [comment]
👀

**Comment by JMurphyWeb** [comment]
Can we not flatten out our tests @nelsonic? There are already easily enough test files to make an argument against that suggestion and i am adding more in deployment branch.

File: server/handlers.js [hunk 1]
```diff
@@ -1,364 +0,0 @@
-const bcrypt = require('bcrypt');
```
Comment (nelsonic) [inline server/handlers.js]: https://www.youtube.com/watch?v=Qy9_lfjQopU
(_thanks for splitting this out @mk4111 ..._)

**Comment by nelsonic** [comment]
@mk4111 thanks _very_ much for doing that #chore 👍 
❤️ ✅ 🎉

**Comment by nelsonic** [comment]
@mk4111 this branch was merged into the `refactor-tests` branch. which is now _ahead_ of `master` ... can you please create _another_ PR to get this into `master` and _assign_ it to me? thanks!

**Comment by mk4111** [comment]
@nelsonic Okidokie!

**Pull request merged**